cmd/ocm-agent/main.go

```diff
@@ -7,6 +7,7 @@ import (
 
 	"github.com/gorilla/mux"
 	"github.com/openshift/ocm-agent/hack"
```

Comment:
Unrelated, change hack to fake?

Reply:
Thanks for your message Ron. I believe this will be removed completely as part of [OSD-8943](https://issues.redhat.com/browse/OSD-8943) is implemented. Please follow up or resolve this discussion